train/retrain.py

```diff
@@ -160,9 +160,10 @@ def create_image_lists(image_dir, testing_percentage, validation_percentage):
     if not file_list:
       tf.logging.warning('No files found')
       continue
-    if len(file_list) < 20:
+    if len(file_list) < 30:
       tf.logging.warning(
           'WARNING: Folder has less than 20 images, which may cause issues.')
```

Comment:
Might need to update the warning message as well. The suggestion in below:
Folder has less than 20 images --> The category folder has less than 30 images